packages/flutter_tools/lib/src/commands/screenshot.dart

```diff
@@ -33,21 +32,20 @@ class ScreenshotCommand extends FlutterCommand {
       aliases: <String>[ 'observatory-url' ], // for historical reasons
       valueHelp: 'URI',
       help: 'The VM Service URL to which to connect.\n'
-          'This is required when "--$_kType" is "$_kSkiaType" or "$_kRasterizerType".\n'
+          'This is required when "--$_kType" is "$_kSkiaType".\n'
           'To find the VM service URL, use "flutter run" and look for '
           '"A Dart VM Service ... is available at" in the output.',
     );
     argParser.addOption(
       _kType,
       valueHelp: 'type',
       help: 'The type of screenshot to retrieve.',
-      allowed: const <String>[_kDeviceType, _kSkiaType, _kRasterizerType],
+      allowed: const <String>[_kDeviceType, _kSkiaType],
       allowedHelp: const <String, String>{
         _kDeviceType: "Delegate to the device's native screenshot capabilities. This "
                       'screenshots the entire screen currently being displayed (including content '
                       'not rendered by Flutter, like the device status bar).',
         _kSkiaType: 'Render the Flutter app as a Skia picture. Requires "--$_kVmServiceUrl".',
-        _kRasterizerType: 'Render the Flutter app using the rasterizer. Requires "--$_kVmServiceUrl."',
```

Comment:
So this PR is only removing the ability to do `flutter screenshot --type=rasterizer --vm-service-url=$URL`?

Reply:
Correct, though the Skia picture type will also eventually become irrelevant as that is only used for SKP debugging